process/sync/baseForkDetector_test.go

```diff
@@ -761,3 +761,67 @@ func TestBasicForkDetector_GetProbableHighestNonce(t *testing.T) {
 	hInfos = bfd.GetHeaders(3)
 	assert.Equal(t, uint64(3), bfd.GetProbableHighestNonce(hInfos))
 }
+
+func TestShardForkDetector_CheckShardBlockValidityShouldWork(t *testing.T) {
```

Comment:
should work in the name not exactly valid for ErrLowerRoundInBlock

Reply:
Changed